.github/workflows/pr-for-updates.yaml

```diff
@@ -0,0 +1,18 @@
+ 
+---
+name: Pull Request on update/* Branch Push
+on:
+  push:
+    branches:
+      - update/*
+jobs:
+  auto-pull-request:
+    name: Pull Request on update/* Branch Push
```

Comment:
Could you incorporate the analog here to this:
https://github.com/networkservicemesh/sdk/blob/c75f462c5a1f99da1e29897109cc0e5cfc163afb/.github/workflows/update-sdk-vppagent.yaml#L8